daemon/tests/happy_path.rs

```diff
@@ -51,6 +50,90 @@ macro_rules! wait_next_state {
     };
 }
 
+macro_rules! confirm {
+    (lock transaction, $id:expr, $maker:expr, $taker:expr) => {
+        $maker
+            .mocks
+            .monitor()
+            .await
+            .confirm_lock_transaction($id)
+            .await;
+        $taker
+            .mocks
+            .monitor()
+            .await
+            .confirm_lock_transaction($id)
+            .await;
+    };
+    (commit transaction, $id:expr, $maker:expr, $taker:expr) => {
+        $maker
+            .mocks
+            .monitor()
+            .await
+            .confirm_commit_transaction($id)
+            .await;
+        $taker
+            .mocks
+            .monitor()
+            .await
+            .confirm_commit_transaction($id)
+            .await;
+    };
+    (refund transaction, $id:expr, $maker:expr, $taker:expr) => {
+        $maker
+            .mocks
+            .monitor()
+            .await
+            .confirm_refund_transaction($id)
+            .await;
+        $taker
+            .mocks
+            .monitor()
+            .await
+            .confirm_refund_transaction($id)
+            .await;
+    };
+    (close transaction, $id:expr, $maker:expr, $taker:expr) => {
+        $maker
+            .mocks
+            .monitor()
+            .await
+            .confirm_close_transaction($id)
+            .await;
+        $taker
+            .mocks
+            .monitor()
+            .await
+            .confirm_close_transaction($id)
+            .await;
+    };
+    (cet, $id:expr, $maker:expr, $taker:expr) => {
+        $maker.mocks.monitor().await.confirm_cet($id).await;
+        $taker.mocks.monitor().await.confirm_cet($id).await;
+    };
+}
+
+macro_rules! expiry {
```

Comment:
nit: `expire`? (to align with macro above? 

Both could also use some simple doc comments, ie. "trigger expiry event in the monitor actor" or something like that